### PR TITLE
Add PDF Metadata Editor

### PR DESCRIPTION
**Pull request by Diego-Ivan**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.pdf_metadata_editor

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.pdf_metadata_editor

File: io.github.diegoivan.pdf_metadata_editor.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          "url": "https://github.com/flatpak/libportal.git",
+          "type": "git",
+          "commit": "09ec040159b754c14610ad02e4de17d0b9bc6612"
+        }
+      ]
+    },
+
+    {
+      "name": "poppler",
+      "buildsystem": "cmake",
```
Comment (hfiguiere) [inline io.github.diegoivan.pdf_metadata_editor.json:48]:
```suggestion
      "buildsystem": "cmake-ninja",
```

File: io.github.diegoivan.pdf_metadata_editor.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        "-DBUILD_MANUAL_TESTS=OFF",
+        "-DENABLE_UTILS=OFF",
+        "-DENABLE_CPP=OFF",
+        "-DENABLE_QT5=OFF",
+        "-DENABLE_QT6=OFF",
+        "-DENABLE_BOOST=OFF"
+      ],
+      "sources": [
+        {
+          "url": "https://gitlab.freedesktop.org/poppler/poppler.git",
```
Comment (hfiguiere) [inline io.github.diegoivan.pdf_metadata_editor.json:64]: please use a release of poppler.

File: io.github.diegoivan.pdf_metadata_editor.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        "-DBUILD_GTK_TESTS=OFF",
+        "-DBUILD_QT5_TESTS=OFF",
+        "-DBUILD_QT6_TESTS=OFF",
+        "-DBUILD_CPP_TESTS=OFF",
+        "-DBUILD_MANUAL_TESTS=OFF",
+        "-DENABLE_UTILS=OFF",
+        "-DENABLE_CPP=OFF",
+        "-DENABLE_QT5=OFF",
+        "-DENABLE_QT6=OFF",
+        "-DENABLE_BOOST=OFF"
```
Comment (hfiguiere) [inline io.github.diegoivan.pdf_metadata_editor.json:60]: I'll assume you don't need the splash backend for poppler.

File: io.github.diegoivan.pdf_metadata_editor.json [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+          "url": "https://gitlab.freedesktop.org/poppler/poppler.git",
+          "type": "git",
+          "commit": "083662257ff626d6d3454a37b70928a62990c7ad"
+        }
+      ]
+    },
+
+    {
+      "name": "pdf-metadata-editor",
+      "builddir": true,
```
Comment (hfiguiere) [inline io.github.diegoivan.pdf_metadata_editor.json:73]:
```suggestion
```

File: io.github.diegoivan.pdf_metadata_editor.json (hunk 3 above)
Comment (Diego-Ivan) [reply io.github.diegoivan.pdf_metadata_editor.json:60]: No, the app does not need it.

**Comment by Diego-Ivan** [comment]
bot, build io.github.diegoivan.pdf_metadata_editor

**Pull request closed without merging**